lib/charty/colors/xyz.rb

```diff
@@ -0,0 +1,125 @@
+require_relative "helper"
+
+require "numo/narray"
```

Comment:
Should we depend on Numo::NArray?

Reply:
numo-narray is the current best solution when we need to do matrix multiplication.